theme/src/styles/content-elements/footer/_index.scss

```diff
@@ -20,196 +20,263 @@
 // check "theme/src/styles/bootstrap/mixins/_index.scss"
 @use '../../bootstrap/mixins' as mixin;
 
-
-// =========================================================
-// === Footer ==============================================
-// =========================================================
+// =================================
+// === Footer  =====================
+// =================================
 .footer {
-  background: $footer-background;
-  border-top: 15px solid $footer-background;
-  margin-top: 50px;
-}
-
-._start-page + .footer {
-  margin: 0;
-}
-
-.footer p {
-  color: $footer-color;
-  font-weight: 400;
-}
+  background: var.$blacklight;
+  padding: 140px 0 60px;
+  font-family: var.$font-family-sans-serif;
+  font-weight: var.$font-weight-medium;
+  font-size: 14px;
+  color: var.$white;
+  -webkit-font-smoothing: antialiased;
+  text-transform: uppercase;
+
+  & h1,
```

Comment:
Do we really need to stile all types of headers?

Reply:
I can use an additional selector as footer__middle. We need to style RTE text in the footer

Reply:
I placed styling to .footer__header.